Content.Shared/SS220/Language/Systems/SharedLanguageSystem.Sanitizer.cs

```diff
@@ -26,10 +27,10 @@ public LanguageMessage SanitizeMessage(EntityUid source, string message)
         if (_cachedMessages.TryGetValue(cacheKey, out var cahcedLanguageMessage))
             return cahcedLanguageMessage;
 
-        List<LanguageNode> nodes = new();
+        List<LanguageNode> nodes = [];
         var defaultLanguage = GetSelectedLanguage(source);
-        if (defaultLanguage == null && !_language.TryGetLanguageById(UniversalLanguage, out defaultLanguage))
-            return new LanguageMessage(nodes, message, this);
+        if (defaultLanguage is null && !_language.TryGetLanguageById(UniversalLanguage, out defaultLanguage))
+            throw new Exception($"\"{UniversalLanguage}\" is invalid universal language id!");
```

Comment:
```suggestion
{
           Log.Fatal($"\"{UniversalLanguage}\" is invalid universal language id!");
           return LanguageMessage.Empty;
 }
```
Exception лишний, ничего кроме твоей системы не ломается. Возвращай просто пустое сообщение